src/containerapp/azext_containerapp/_help.py

```diff
@@ -2337,102 +2337,3 @@
       text: |
           az containerapp label-history show -n my-containerapp -g MyResourceGroup --label LabelName
 """
```

Comment:
1. Please update `\src\containerapp\azext_containerapp\azext_metadata.json`, set `"azext.minCliCoreVersion": "2.79.0"`

2. Update `HISTORY.rst`:
```
* Update azure cli dependency version >= "2.79.0"
* `az containerapp env http-route-config/premium-ingress`: Remove GA commands which exist in azure-cli version 2.79.0
```

Reply:
I updated the branch